packages/cdk-graph-plugin-diagram/src/internal/graphviz/entities/nodes.ts

```diff
@@ -109,25 +107,15 @@ export class ImageNode extends Node {
   }
 
   /** Resizes the node based on image and label dimensions */
-  resize(baseHeight?: number): void {
-    if (baseHeight == null) {
-      baseHeight = (this.attributes.get("height") || 1) as number;
-    }
-
+  resize(): void {
     const image = this.image;
 
     if (image) {
-      const labelLines = this.label.split("\n").length;
-
+      this.attributes.set("width", 0.5);
       this.attributes.set("labelloc", "b");
-
-      this.attributes.set(
-        "height",
-        baseHeight + labelLines * LABEL_LINE_HEIGHT
-      );
     } else {
       this.attributes.set("labelloc", "c");
-      this.attributes.set("height", baseHeight);
+      this.attributes.set("penwidth", 0.25);
     }
```

Comment:
Remember with our dynamic height based on "label lines height" the rendering was jank. If I remember correctly there was overlap and incorrect edge placement. Did you get that resolved or is no longer an issue?

Reply:
Looks like the image based node labeling is "right aligned" (instead of centered), and the edge connecting to image nodes is not centered as well - so to my above comment, this code was to correct those alignment issues.

<img width="882" alt="image" src="https://github.com/aws/aws-pdk/assets/464119/6ddf8234-ea32-446f-8e07-d6ba7ced34ef">